openmdao/utils/variable_table.py

```diff
@@ -203,11 +203,8 @@ def write_source_table(source_dicts, out_stream):
     if out_stream is None:
         return
 
-    # use tabulate if we are in a notebook, have tabulate and are using the default out_stream
-    if notebook and tabulate and out_stream is _DEFAULT_OUT_STREAM:
-        use_tabulate = True
-    else:
-        use_tabulate = False
+    # use table_builder if we are in a notebook and are using the default out_stream
+    use_html = notebook and out_stream is _DEFAULT_OUT_STREAM
```

Comment:
a nit, but would be nice to be consistent with either `use_table_builder` or `use_html`